### PR TITLE
Fixed layout

### PR DESCRIPTION
**Pull request by ismarS**
As referenced in https://github.com/yahoo/pure-site/issues/308 

Replaced the classes to reflect those in `pure.css` and `grids-responsive.css`

I think there might be a better way to handle `grids-responsive.css` inclusion, something closer to how `pure.css` gets implemented. Perhaps a possible edit?

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by ismarS** [comment]
I signed the CLA, should be fine now.

**Comment by redonkulus** [comment]
Thanks! Will pull this down and check locally.

**Comment by redonkulus** [comment]
@ismarS So I noticed that even though the units are the same, they size of each image region is not the same for your changes vs the ones on the site now. 

For example, if you make the size of the browser over 1024px (basically the large size) and have your changes open in browser tab and the [page in production](http://purecss.io/layouts/gallery/). That the width of the first image does not line up. Changing it from `pure-u-lg-1-4` to `pure-u-lg-2-4` fixes it but then you have to adjust the images that follow. 

Also, checking when the browser width between 770x and 1024px (medium) does not match up. 

Overall, I think you are close but just need to adjust the units a bit to match the existing layout.

**Comment by ismarS** [comment]
I see what you mean, however, I THINK (even though it looks kinda bad) that this is intentional (?) In order to demonstrate how different classes size images differently (showing 1/4 vs 3/4 and in the next row it's 3x1/3). 

On the other hand, what's the point if it looks bad - so in order to fix this, it's just necessary to change the `pure-u-lg-1-4` to `pure-u-lg-1-3` for the left image and for the right image `pure-lg-3-4` to `pure-lg-2-3`

On the other hand between 774 and 1024px (medium) in order to align we just need to switch from `pure-u-md-1-3` to `pure-u-md-1-2` and on the other from `pure-u-md-2-3` to `pure-u-md-1-2` and they will align. This is a fix only if they are MEANT to align. Which I, personally, think they should :smiley:

Oh, and also, this problem with the medium sizes is present on live as well. 

Also, switching to 1/2 for the first row of images remedies the issue from https://github.com/yahoo/pure-site/issues/288 a bit as well. 

Let me know if you think we should do these changes so that the images align, and I'l _git_ to it :wink:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by redonkulus** [comment]
Ya I would say fix it if you can, especially if it helps the distortion.

File: views/pages/layouts/examples/gallery.handlebars [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{setTitle    "Photo Gallery"}}
 {{setSubTitle "A layout example that shows off a responsive photo gallery."}}
 {{setTags     "base" "grids" "forms" "buttons" "menus"}}
-{{addLocalCSS "/css/layouts/gallery-grid.css" hasOldIE=true}}
+{{addRemoteCSS "https://yui-s.yahooapis.com/pure/0.6.0/grids-responsive-min.css"}}
```
Comment (redonkulus) [inline views/pages/layouts/examples/gallery.handlebars]: For the 0.6.0 part, can you use the `{{pure.version}}` variable so the version gets updated as pure releases new versions?

Comment (ismarS) [inline views/pages/layouts/examples/gallery.handlebars]: @redonkulus Got it. I thought there might be a better way of handling this.

**Comment by ismarS** [comment]
@redonkulus I'll get on it, either today or tomorrow, but I'll apply the fixes soon enough 😉

**Comment by redonkulus** [comment]
Closing in favor of https://github.com/yahoo/pure-site/pull/350

**Pull request closed without merging**